dev-lang/ispc/ispc-1.14.0.ebuild

```diff
@@ -57,3 +59,7 @@ src_install() {
 		doins -r "${BUILD_DIR}"/examples/*
```

Comment:
This might have been discussed with someone else before, but how big are the examples? If they're small, we'll just install them unconditionally.

Reply:
The example directory is 78 MB. So it is not that big but not that small either.

Reply:
This is probably enough to justify the USE flag. Thanks!